pkg/core/blockchain_core_test.go

```diff
@@ -219,7 +219,7 @@ func TestBlockchain_InitWithIncompleteStateJump(t *testing.T) {
 		checkNewBlockchainErr(t, func(c *config.Config) {
 			boltCfg(c)
 			c.ApplicationConfiguration.RemoveUntraceableBlocks = false
-		}, bcSpout.dao.Store, "state jump was not completed, but P2PStateExchangeExtensions are disabled or archival node capability is on")
+		}, bcSpout.dao.Store, "P2PStateExchangeExtensions can be enabled either on MPT-complete node")
```

Comment:
either?

Reply:
I've not copy-pasted the whole string, it's _very_ long.

Reply:
Sure, but `either` can be removed, there's only one option now.